src/Microsoft.DotNet.Build.Tasks.Feed/src/model/TargetFeedConfig.cs

```diff
@@ -71,7 +71,7 @@ public TargetFeedConfig(TargetFeedContentType contentType,
             bool isolated = false, 
             bool @internal = false, 
             bool allowOverwrite = false, 
-            SymbolTargetType symbolTargetType = SymbolTargetType.None, 
+            SymbolPublishVisibility symbolTargetType = SymbolPublishVisibility.None, 
```

Comment:
nit: there are a bunch of places where the type name got upated but not the variable name.

```suggestion
            SymbolPublishVisibility symbolPublishVisibility = SymbolPublishVisibility.None, 
```

Reply:
These are two different types

Reply:
Sure, the point being that the variable name is still symbolTargetType which makes the numeric comparisons in https://github.com/dotnet/arcade/pull/15055#discussion_r1750820143 a bit confusing.